retroarch.c

```diff
@@ -35194,6 +35210,13 @@ static bool retroarch_parse_input_and_config(
          }
       }
    }
+   verbosity_enabled = verbosity_is_enabled();
+   
+   if (verbosity_enabled)
+      rarch_log_file_init(
```

Comment:
This makes me uncomfortable. When you call `rarch_log_file_init()` here, it is likely that none of the arguments will be defined (because we haven't yet loaded a config file)

The arguments should be set explicitly instead (`true`/`false` and empty strings) depending upon command line options, with a comment explaining the implications of using these values.

Reply:
Thanks this is true, I'll now pass "true", "false", and '\0' for these (the latter two won't actually be used anyway as long as the file log override is set).